### ISSUE TITLE
Error Code: 1006 with Browser Websocket Client

### ISSUE THREAD

**Issue by ChaitanyaYeole02** [villekr/ocpp-asgi#18]
Hello @villekr ,
I am trying to connect a browser websocket client 
```
https://chrome.google.com/webstore/detail/browser-websocket-client/mdmlhchldhfnfnkfmljgeinlffmdgkjo/related?hl=en
```
I am receiving Error Code: 1006  while doing the same.
Can you please look into this?
I am a bit confused, it this due to uvicorn implementation or something else?

**Comment by ChaitanyaYeole02** [comment]
This error is solved in the uvicorn library is updated to 0.20.0, please check and update it in the requirements file

**Comment by villekr** [comment]
Good investigation. I took a look if upgrading to uvicorn 0.20.0 would be ok. But apparently it has some issue with subprotocol handling. I opened a discussion (https://github.com/encode/uvicorn/discussions/1777) whether that's a bug and before upgrading from current (0.18.3) version that uvicorn issue must be solved somehow.

ocpp_asgi doesn't currently return subprotocol during websocket.accept. I'll fix that once above uvicorn issue is resolved.
